hydra-access-controls/lib/hydra-access-controls.rb

```diff
@@ -2,6 +2,7 @@
 require 'active-fedora'
 require 'blacklight'
 require 'cancan'
+require 'action_view' # https://github.com/haml/haml/issues/695
```

Comment:
Haml 5.0 is out with this fix https://github.com/haml/haml/commit/f87e2cbdf1ba8d850ba50c9f81719eebfb887c14 so 👍  to close.